deltacat/tests/compute/compact_partition_rebase_test_cases.py

```diff
@@ -65,20 +67,68 @@ class RebaseCompactionTestCaseParams(BaseCompactorTestCase):
             ],
             names=["pk_col_1", "sk_col_1", "sk_col_2", "col_1"],
         ),
-        expected_terminal_compact_partition_result=pa.Table.from_arrays(
+        expected_terminal_compact_partition_result=pa.Table.from_arrays([]),
+        expected_terminal_exception=None,
+        expected_terminal_exception_message=None,
+        do_create_placement_group=False,
+        records_per_compacted_file=DEFAULT_MAX_RECORDS_PER_FILE,
+        hash_bucket_count=DEFAULT_HASH_BUCKET_COUNT,
+        read_kwargs_provider=None,
+        drop_duplicates=True,
+        skip_enabled_compact_partition_drivers=[CompactorVersion.V1],
+    ),
+    "2-rebase-sort": RebaseCompactionTestCaseParams(
+        primary_keys={"pk_col_1"},
+        sort_keys=[
+            SortKey.of(key_name="sk_col_1"),
+            SortKey.of(key_name="sk_col_2"),
+        ],
+        partition_keys=[PartitionKey.of("region_id", PartitionKeyType.INT)],
+        partition_values=["1"],
+        input_deltas=pa.Table.from_arrays(
             [
                 pa.array([str(i) for i in range(10)]),
-                pa.array([i for i in range(20, 30)]),
+                pa.array([i for i in range(0, 10)]),
                 pa.array(["foo"] * 10),
-                pa.array([i / 10 for i in range(40, 50)]),
+                pa.array([i / 10 for i in range(10, 20)]),
+                pa.array(dt.datetime(year, 1, 1) for year in range(2000, 2010)),
+            ],
+            names=[
+                "pk_col_1",
+                "sk_col_1",
+                "sk_col_2",
+                "col_1",
+                DW_LAST_UPDATED_COLUMN_NAME,
+            ],
+        ),
+        input_deltas_delta_type=DeltaType.UPSERT,
+        # dw_last_update is in ascending order in the input table.
+        # Expect descending sort on dw_last_updated for each hash bucket.
+        # Since there is only one hash bucket, the order of input rows should be reversed.
+        rebase_expected_compact_partition_result=pa.Table.from_arrays(
+            [
+                pa.array([str(i) for i in reversed(range(10))]),
+                pa.array([i for i in reversed(range(0, 10))]),
+                pa.array(["foo"] * 10),
+                pa.array([i / 10 for i in reversed(range(10, 20))]),
+                pa.array(
+                    dt.datetime(year, 1, 1) for year in reversed(range(2000, 2010))
+                ),
+            ],
+            names=[
+                "pk_col_1",
+                "sk_col_1",
+                "sk_col_2",
+                "col_1",
+                DW_LAST_UPDATED_COLUMN_NAME,
             ],
-            names=["pk_col_1", "sk_col_1", "sk_col_2", "col_1"],
         ),
+        expected_terminal_compact_partition_result=pa.Table.from_arrays([]),
         expected_terminal_exception=None,
         expected_terminal_exception_message=None,
         do_create_placement_group=False,
         records_per_compacted_file=DEFAULT_MAX_RECORDS_PER_FILE,
```

Comment:
max records per file is 4M but your test case doesn't have more than 4M records. So, it only tests sorting within a single file but not across multiple files.

Reply:
Would you recommend reducing `DEFAULT_MAX_RECORDS_PER_FILE` to a number smaller than than the `input_deltas` table?

Reply:
(At least, for the new test case?)